tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/GoLanguageSpecificChecks.cs

```diff
@@ -178,5 +161,19 @@ public async Task<PackageCheckResponse> ValidateChangelog(string packagePath, bo
         var packageName = await GetSDKPackageName(repoRoot, packagePath, cancellationToken);
         return await _commonValidationHelpers.ValidateChangelog(packageName, packagePath, fixCheckErrors, cancellationToken);
     }
+
+    public async Task<bool> RunAllTests(string packagePath, CancellationToken ct = default)
+    {
+        try
+        {
+            var result = await _processHelper.Run(new ProcessOptions(compilerName, ["test", "-v", "-timeout", "1h", "./..."], compilerNameWindows, ["test", "-v", "-timeout", "1h", "./..."], workingDirectory: packagePath), ct);
```

Comment:
nit: result is not returned.

Reply:
Oof, thanks for the catch. I thought processHelper.Run threw on failed exit codes - apparently it doesn't so I did need to do an exit code check.